js/src/product-feed/issues-table-card/issues-solved.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { capitalize } from 'lodash/string';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalText as Text, Dashicon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ISSUE_TYPE_PRODUCT } from '.~/constants';
+import useActiveIssueType from '.~/hooks/useActiveIssueType';
+
+/**
+ * This component renders a message when no issues of the
+ * active issue type are pending to solve.
+ *
+ * @return {JSX.Element} The component
+ */
+const IssuesSolved = () => {
+	const issueType = useActiveIssueType();
+	const otherIssueType =
+		issueType === ISSUE_TYPE_PRODUCT ? 'account' : 'product';
+
+	const map = {
+		otherIssueTab: <strong>{ capitalize( otherIssueType ) } Issues</strong>,
+		otherIssueType: <span>{ otherIssueType }</span>,
+	};
```

Comment:
❓ 
It should only apply the bold font tag here. Whenever possible, it'd be better to place the complete sentences in strings that need to be translated and avoid composing them programmatically. if it's still necessary, use `sprintf` instead.

Reply:
Updated here:

https://github.com/woocommerce/google-listings-and-ads/pull/1196/commits/f72fa4f23758c31147499fe1a2607a7b805224c5